tools/Vrops.py

```diff
@@ -79,32 +79,29 @@ def get_vcenter_adapter(self, target, token):
     def get_nsxt_adapter(self, target, token):
         return self.get_adapter(target, token, adapterkind="NSXTAdapter", adapter_obj=NSXTAdapterInstance)
 
-    def get_resources(self, target: str, token: str, uuids: list, adapterkind: str, resourcekinds: list, resource_obj,
-                      data_receiving=False) -> (list, int):
+    def get_resources(self, target: str, token: str,
+                      uuids: list,
+                      adapterkind: str,
+                      resourcekinds: list,  # Array of resource kind keys
+                      resource_class,  #
+                      resource_status: list = None  # Array of resource data collection stats
+                      ) -> (list, int):
         logger.debug(f'Getting {resourcekinds} from {target}')
         url = "https://" + target + "/suite-api/api/resources/bulk/relationships"
         querystring = {
-            'pageSize': '100000'
+            'pageSize': 10000
         }
+        resource_status_array = [] if not resource_status else resource_status
         payload = {
             "relationshipType": "DESCENDANT",
             "resourceIds": uuids,
             "resourceQuery": {
                 "adapterKind": [adapterkind],
                 "resourceKind": resourcekinds,
-                "resourceStatus": ["DATA_RECEIVING"]
-            },
-            "hierarchyDepth": 5
-        } if data_receiving else {
-            "relationshipType": "DESCENDANT",
-            "resourceIds": uuids,
-            "resourceQuery": {
-                "adapterKind": [adapterkind],
-                "resourceKind": resourcekinds
+                "resourceStatus": resource_status_array
```

Comment:
Why changing this? So far I can only see `["DATA_RECEIVING"]` being used, which is now in the code multiple times.

Reply:
The same argument. It is a preparation for configurability.

Reply:
So it's going to be in the configfile soon?

Reply:
Yes. This will lead to comprehensive changes.

Reply:
Therefore I will open a new PR

Reply:
Alright, created #168 for tracking.